lithops/standalone/backends/ibm_vpc/ibm_vpc.py

```diff
@@ -80,6 +80,66 @@ def __init__(self, ibm_vpc_config, mode):
         msg = COMPUTE_CLI_MSG.format('IBM VPC')
         logger.info(f"{msg} - Region: {self.region}")
 
+    def _create_ssh_key(self):
+
+        DEFAULT_KEY_NAME = "default-ssh-key"
+        def _generate_keypair(keyname):
+            """Returns newly generated public ssh-key's contents and private key's path"""
+            home = str(Path.home())
+            filename = f"{home}{os.sep}.ssh{os.sep}id.rsa.{keyname}"
+            try:
+                os.remove(filename)
+            except Exception:
+                pass
+
+            os.system(f'ssh-keygen -b 2048 -t rsa -f {filename} -q -N ""')
+            logger.debug(f"\n\n\033[92mSSH key pair been generated\n")
+            logger.debug(f"private key: {os.path.abspath(filename)}")
+            logger.debug(f"public key {os.path.abspath(filename)}.pub\033[0m")
+            with open(f"{filename}.pub", "r") as file:
+                ssh_key_data = file.read()
+            ssh_key_path = os.path.abspath(filename)
+            return ssh_key_data, ssh_key_path
+
+        def _get_ssh_key(ibm_vpc_client, name):
+            """Returns ssh key matching specified name, stored in the VPC associated with the vpc_client"""
+            for key in ibm_vpc_client.list_keys().result["keys"]:
+                if key["name"] == name:
+                    return key
+
+        def _register_ssh_key(ibm_vpc_client, resource_group_id):
+
+            keyname = DEFAULT_KEY_NAME
```

Comment:
I would put `keyname=self.vpc_name`, this way each user will have a different `keyname`, since the "keys space" (https://cloud.ibm.com/vpc-ext/compute/sshKeys) is a space shared for all the users.

Reply:
@JosepSampe done